vowpalwabbit/multiclass.cc

```diff
@@ -13,10 +13,19 @@
 
 namespace MULTICLASS
 {
+  label_t::label_t() { reset_to_default(); }
```

Comment:
maybe `set_to_default` instead? it's counter intuitive for a constructor to `reset`

Reply:
Ah set is much better, thanks!